ddl/table.go

```diff
@@ -130,19 +127,23 @@ func onDropTable(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 	return ver, errors.Trace(err)
 }
 
-func splitTableRegion(store kv.Storage, tableID int64) error {
+type splitableStore interface {
```

Comment:
Why add it?

Reply:
I don't add it, I only move it from `splitTableRegion` to here.

Reply:
But it still appears in `splitTableRegion`.

Reply:
Should we remove the code at line 135~137?